cudax/include/cuda/experimental/__detail/config.cuh

```diff
@@ -24,7 +24,7 @@
 // Debuggers do not step into functions marked with __attribute__((__artificial__)).
 // This is useful for small wrapper functions that just dispatch to other functions and
 // that are inlined into the caller.
-#if _CCCL_HAS_ATTRIBUTE(__artificial__) && !defined(_CCCL_CUDACC)
```

Comment:
@ericniebler I am not sure whether this is the right condition. _CCCL_CUDACC is defined for every cuda compiler. Do we want this only for nvcc and nvhpc?

Reply:
probably